src/Countries/Australia.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Australia extends Country
+{
+    public function countryCode(): string
+    {
+        return 'au';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'New Year\'s Day' => '01-01',
+            'Australia Day' => '01-26',
+            'Anzac Day' => '04-25',
+            'Christmas Day' => '12-25',
+            'Boxing Day' => '12-26',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))->setTimezone('Australia/Sydney');
```

Comment:
Could you use `$this->easter($year)` instead? Available after rebasing from this repo's main branch.

Reply:
This is done.